DuckDuckGo/NavigationBar/View/NavigationBarViewController.swift

```diff
@@ -251,7 +251,7 @@ final class NavigationBarViewController: NSViewController {
 
     @IBAction func refreshOrStopAction(_ sender: NSButton) {
         guard let selectedTabViewModel = tabCollectionViewModel.selectedTabViewModel else {
-            os_log("%s: Selected tab view model is nil", type: .error, className)
+            Logger.navigation.error("Selected tab view model is nil")
```

Comment:
are we okay with skipping className from these logs?

Reply:
If subsystem and categories are correct the class name shouldn't be necessary, If needed can be re-added on demand, you generally don't see class names, frameworks names in the log content from Apple.